src/cpuad-updater/grafana/dashboard-template.json

```diff
@@ -19,7 +19,6 @@
         "editable": true,
         "fiscalYearStartMonth": 0,
         "graphTooltip": 0,
-        "id": 4,
```

Comment:
In the other PR I've added code that removes it so you don't have to remember about cutting it out every time template changes